dspace-server-webapp/src/main/java/org/dspace/app/rest/ItemOwningCollectionUpdateRestController.java

```diff
@@ -97,7 +98,7 @@ public CollectionRest move(@PathVariable UUID itemUuid, HttpServletResponse resp
         if (targetCollection == null) {
             return null;
         }
-        return converter.fromModel(targetCollection);
+        return converter.toRest(targetCollection, Projection.DEFAULT);
```

Comment:
Why is this a hardcoded Projection instead of `utils.obtainProjection()`?  Just noticing that most Controllers call `utils.obtainProjection` while this one hardcodes it to `DEFAULT`.